docs/Kb.md

```diff
@@ -0,0 +1,8 @@
+This file documents the places the `Kb` (Knowledgebase) are used within ZAP and zap-extensions. (`org.parosproxy.paros.core.scanner.Kb`)
```

Comment:
This should say “is” used…will tweak in the morning

Reply:
Ping.

Reply:
Thanks will tackle it shortly ;)